frontend/src/__tests__/cypress/cypress/e2e/home/homeUtils.ts

```diff
@@ -13,3 +17,7 @@ export const initHomeIntercepts = (config: HandlersProps = {}): void => {
   };
   cy.interceptOdh('GET /api/config', mockDashboardConfig(dashboardConfig));
```

Comment:
FWIW, you can just type your utility with the dashboardConfig settings for the mock... saving you the dupe effort for your type.

Since you're trying to mimic the first argument to this function, you can just set your function to take the same input.

```
export const initHomeIntercepts = (
  config: Parameters<typeof mockDashboardConfig>[0] = {},
): void => {
```

Or if you want, you can go one step further:
```
export const initHomeIntercepts = (...args: Parameters<typeof mockDashboardConfig>): void => {
  cy.interceptOdh('GET /api/config', mockDashboardConfig(...args));
};
```